### ISSUE TITLE
git sync issue

### ISSUE THREAD

**Issue by jeanchristopheMICHEL** [ArchGPT/insomnium#91]
### Expected Behavior

When I use the (re-)new git sync menu (from 0.2.3-a), with Git environnement well done, when I use "git pull" after a "git fetch", I must see requests collection.

### Actual Behavior

After "git fetch" action, I can see commit history, all branches, but there is no request
![Screen Shot 11-03-23 at 11 23 AM](https://github.com/ArchGPT/insomnium/assets/140072308/12462ad1-b30d-42e6-a4ed-231938a205a4)

### Reproduction Steps

1. Define git synch 
2. create 2 simple requests
3. git comit
4. git push 
5. confirm this commit is present from gitlab interface

From an other laptop 
6. define git synch
7. git pull

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/ArchGPT/insomnium/issues) for this problem.

### Insomnium Version

0.2.3-a

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Entreprise 22H2

### Installation method

download .exe

### Last Known Working Insomnium version

0.2.1

**Comment by LCLLL** [comment]
Yeah, the same thing happened to me. When i git clone one that already exists "Collections" project, it will auto group into "Documents", and show some error like attach image.
![Uploading iShot_2023-11-06_14.36.10.png…]()

**Comment by stehlifn** [comment]
+1

**Comment by alex-graz** [comment]
+1

**Comment by cextremerar** [comment]
+1

**Comment by educbraga** [comment]
I'm not sure, but I think Insomnium searches for .insomnium folders, not .insomnia.

**Comment by ArturVRSampaio** [comment]
+1

**Comment by trbtm** [comment]
I had the same problems. They were caused by two collections with the same name in the git repository. I reverted the commits adding another collection and used the `Git Clone` functionality and that solved my issues. Maybe this helps someone.
